components/courses/app/finders/courses/test_task/all_test_tasks.rb

```diff
@@ -0,0 +1,34 @@
+module Courses
+  class TestTask < ApplicationRecord
+    class AllTestTasks < ApplicationFinder
+      def initialize(season)
+        @season = season
+      end
+
+      def call
+        base_scope = season_test_tasks
+                         .includes(student: :user)
+
+        @test_tasks ||= if exists?
+                          base_scope.includes(mentor: :user)
+                        else
+                          base_scope
+                        end
+      end
+
+      private
+
+      attr_reader :season
+
+      def exists?
```

Comment:
The name of the method is not clear.
Who `exists?` Maybe `test_ask_without_menthor_exists?`?